ocfweb/account/register.py

```diff
@@ -63,22 +74,44 @@ def request_account(request: HttpRequest) -> Union[HttpResponseRedirect, HttpRes
 
     real_name = directory.name_by_calnet_uid(calnet_uid)
 
+    association_choices = []
+    if not existing_accounts or calnet_uid in TESTER_CALNET_UIDS:
+        association_choices.append((calnet_uid, real_name))
+    for group_oid, group in eligible_group_signatories.items():
+        association_choices.append((group_oid, group['name']))
+
     if request.method == 'POST':
-        form = ApproveForm(request.POST)
+        form = ApproveForm(request.POST, association_choices=association_choices)
         if form.is_valid():
-            req = NewAccountRequest(
-                user_name=form.cleaned_data['ocf_login_name'],
-                real_name=real_name,
-                is_group=False,
-                calnet_uid=calnet_uid,
-                callink_oid=None,
-                email=form.cleaned_data['contact_email'],
-                encrypted_password=encrypt_password(
-                    form.cleaned_data['password'],
-                    RSA.importKey(CREATE_PUBLIC_KEY),
-                ),
-                handle_warnings=NewAccountRequest.WARNINGS_WARN,
-            )
+            is_group_account = form.cleaned_data['account_association'] != calnet_uid
+            if is_group_account:
+                req = NewAccountRequest(
+                    user_name=form.cleaned_data['ocf_login_name'],
+                    real_name=real_name,
+                    is_group=True,
+                    calnet_uid=None,
+                    callink_oid=form.cleaned_data['account_association'],
```

Comment:
does django validate server-side that this must be one of the choices? (in other words, can I manually craft a request to register an org as an arbitrary oid?